README.md

```diff
@@ -14,8 +14,33 @@ has only read-access to the database, a writer has read- and write-access.
 A database can be accessed either by any number of readers or by exactly one
 writer at the same time.
 
+## Installing the required libraries.
+
+### On Debian/Ubuntu
+
+```
+sudo apt-get update
+sudo apt-get install libgdbm-dev
```

Comment:
```suggestion
sudo apt update
sudo apt install libgdbm-dev
```

I think the `apt` is better rater than `apt-get`. You can see <https://itsfoss.com/apt-vs-apt-get-difference/> for the context.